src/main/java/core/basesyntax/Constants.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public final class Constants {
```

Comment:
You are not using this values in multiple classes. That's why it's better not to hold separate class for constants